enrichgraph/enrichgraph.go

```diff
@@ -0,0 +1,23 @@
+package enrichgraph
+
+import (
+	"github.com/jfrog/jfrog-client-go/xray"
+	"github.com/jfrog/jfrog-client-go/xray/services"
+)
+
+const (
+	EnrichMinimumVersionXray = "3.90"
```

Comment:
Not used... probably you forgot to change after copy/paste...

Reply:
I'm not sure what will be the xray version for it .. what can I do about it?